### PR TITLE
create a shared_example for the command class

### PR DESCRIPTION
**Pull request by MaximilianMeister**

### CODE REVIEW

**Comment by tboerger** [comment]
Subject should stay within the spec definition as it's related to all of the specs.

**Comment by MaximilianMeister** [comment]
> Subject should stay within the spec definition as it's related to all of the specs.

as it is done like that in request's shared example we agreed to keep it like that.

and we agreed to just create an empty class for the node commands that are exceptions which inherits from `Action`, then it is more visible and we don't have to treat them as exceptional in the specs anymore

**Comment by tboerger** [comment]
> as it is done like that in request's shared example we agreed to keep it like that.
> 
> and we agreed to just create an empty class for the node commands that are exceptions which inherits from Action, then it is more visible and we don't have to treat them as exceptional in the specs anymore

:+1:

**Comment by MaximilianMeister** [comment]
> as it is done like that in request's shared example we agreed to keep it like that.

and we agreed to just create an empty class for the node commands that are exceptions which inherits from Action, then it is more visible and we don't have to treat them as exceptional in the specs anymore

@tboerger done

**Comment by rsalevsky** [comment]
+1

File: spec/support/command_examples.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+shared_examples "a command class" do |with_body|
+  let!(:superclass_description) do
+    self.class.superclass_metadata[:description]
+  end
+  let(:superclass) do
```
Comment (tboerger) [inline spec/support/command_examples.rb]: Please add an empty line above that, than we can merge it :)

Comment (MaximilianMeister) [inline spec/support/command_examples.rb]: @tboerger done

**Pull request merged**